cypress/e2e/marc/marc-bibliographic/edit-marc-bib/automated-linking/edit-marc-bib-auto-linking-with-multiple-matching-records.cy.js

```diff
@@ -0,0 +1,146 @@
+import Logs from '../../../../../support/fragments/data_import/logs/logs';
+import InventoryInstance from '../../../../../support/fragments/inventory/inventoryInstance';
+import MarcAuthority from '../../../../../support/fragments/marcAuthority/marcAuthority';
+import QuickMarcEditor from '../../../../../support/fragments/quickMarcEditor';
+import TopMenu from '../../../../../support/fragments/topMenu';
+import Users from '../../../../../support/fragments/users/users';
+import getRandomPostfix from '../../../../../support/utils/stringTools';
+import InventoryInstances from '../../../../../support/fragments/inventory/inventoryInstances';
+import InventoryViewSource from '../../../../../support/fragments/inventory/inventoryViewSource';
+
+describe('MARC -> MARC Bibliographic -> Edit MARC bib -> Automated linking', () => {
+  const testData = {
+    tags: {
+      tag245: '245',
+      tag650: '650',
+    },
+    fieldContents: {
+      tag245Content: 'updated 245',
+      tag650Content: '$a Oratory. $2 fast $0 sh85095299C389476',
+    },
+  };
+
+  const marcFiles = [
+    {
+      marc: 'marcBibFileForC389476.mrc',
+      fileName: `testMarcFile.${getRandomPostfix()}.mrc`,
+      jobProfileToRun: 'Default - Create instance and SRS MARC Bib',
+      numOfRecords: 1,
+    },
+    {
+      marc: 'marcAuthFileForC389476-1.mrc',
+      fileName: `testMarcFile.${getRandomPostfix()}.mrc`,
+      jobProfileToRun: 'Default - Create SRS MARC Authority',
+      numOfRecords: 1,
+    },
+    {
+      marc: 'marcAuthFileForC389476-2.mrc',
+      fileName: `testMarcFile.${getRandomPostfix()}.mrc`,
+      jobProfileToRun: 'Default - Create SRS MARC Authority',
+      numOfRecords: 1,
+    },
+  ];
+
+  let userData = {};
+  const linkableField = 650;
+  const createdRecordIDs = [];
+
+  before(() => {
+    cy.createTempUser([
+      Permissions.inventoryAll.gui,
+      Permissions.uiQuickMarcQuickMarcBibliographicEditorAll.gui,
+      Permissions.uiMarcAuthoritiesAuthorityRecordView.gui,
+      Permissions.uiQuickMarcQuickMarcBibliographicEditorCreate.gui,
+      Permissions.uiQuickMarcQuickMarcAuthorityLinkUnlink.gui,
+    ]).then((createdUserProperties) => {
+      userData = createdUserProperties;
+
+      cy.loginAsAdmin().then(() => {
+        marcFiles.forEach((marcFile) => {
+          cy.visit(TopMenu.dataImportPath);
+          DataImport.verifyUploadState();
+          DataImport.uploadFileAndRetry(marcFile.marc, marcFile.fileName);
+          JobProfiles.waitLoadingList();
+          JobProfiles.search(marcFile.jobProfileToRun);
+          JobProfiles.runImportFile();
+          JobProfiles.waitFileIsImported(marcFile.fileName);
+          Logs.checkStatusOfJobProfile('Completed');
+          Logs.openFileDetails(marcFile.fileName);
+          for (let i = 0; i < marcFile.numOfRecords; i++) {
+            Logs.getCreatedItemsID(i).then((link) => {
+              createdRecordIDs.push(link.split('/')[5]);
```

Comment:
Please link marc authority records to marc bib record

Reply:
Hi Tetiana, thanks for comment, I talked to QA and that requirement is redundant, please see the updated test: https://foliotest.testrail.io/index.php?/cases/view/389476